nixos/modules/services/x11/desktop-managers/default.nix

```diff
@@ -72,7 +72,9 @@ in
         apply = map (d: d // {
           manage = "desktop";
           start = d.start
-          + optionalString (needBGCond d) ''\n\n
```

Comment:
The proper solution would probably be to ensure that `d.start` is always newline terminated,
but I don't have the time and energy to follow that rabbit hole.